cli.js

```diff
@@ -1,30 +1,27 @@
 #!/usr/bin/env node
 'use strict';
-const fs = require('fs');
-const path = require('path');
 const arrify = require('arrify');
 const meow = require('meow');
 const getStdin = require('get-stdin');
-const pathExists = require('path-exists');
-const Imagemin = require('imagemin');
+const imagemin = require('imagemin');
+const stripIndent = require('strip-indent');
 
 const cli = meow(`
 	Usage
-	  $ imagemin <file> <directory>
-	  $ imagemin <directory> <output>
+	  $ imagemin <path|glob> ... --out-dir build
 	  $ imagemin <file> > <output>
 	  $ cat <file> | imagemin > <output>
 	  $ imagemin [--plugin <plugin-name>...] ...
 
 	Options
+	  -O, --optimizationLevel <number>  Optimization level between 0 and 7
 	  -P, --plugin                      Override the default plugins
 	  -i, --interlaced                  Interlace gif for progressive rendering
-	  -o, --optimizationLevel <number>  Optimization level between 0 and 7
+	  -o, --out-dir                     Output directory
 	  -p, --progressive                 Lossless conversion to progressive
```

Comment:
I think we should do what we did in gulp-imagemin and drop all the plugin options here. We can rather introduce ability to supply plugin options directly to the plugins in another PR.